src/lib/swapper/swappers/CowSwapper/getCowSwapMinMax/getCowSwapMinMax.ts

```diff
@@ -13,14 +13,20 @@ import {
 import { selectSellAssetUsdRate } from 'state/zustand/swapperStore/amountSelectors'
 import { swapperStore } from 'state/zustand/swapperStore/useSwapperStore'
 
+import { isCowswapSupportedChainId } from '../utils/utils'
+
 export const getCowSwapMinMax = (
```

Comment:
Product Q but we probably want to split `MIN_COWSWAP_VALUE_USD` into ETH and Gnosis min. values.
20 bucks makes sense on mainnet, where the buy asset fees would eat a huge part of the swap, but not on Gnosis which is super cheap, see:

<img width="518" alt="image" src="https://github.com/shapeshift/web/assets/17035424/195385ee-3cb4-4f50-bf0e-a63e29b44199">

Reply:
Confirmed with @reallybeard we can roll with a sub-dollar amount i.e 0.5$ for Gnosis. Will tackle as a follow-up not to keep this one stale.

Reply:
Thought about that too - will post as a follow-up